src/arbitraries/Arbitrary.ts

```diff
@@ -44,7 +46,34 @@ export abstract class Arbitrary<A> {
   }
 
   /**
-   * The special cases for this arbitrary, which can be used during sampling to give higher weight to certain elements.
+   * Returns a sample of picks of a given size without replacements.
+   *
+   * While this function doesn't pick corner cases, each pick is not necessarily independent
+   * from the others: different arbitraries might choose to use smarter search strategies to
+   * maximize the probability of falsifying a property, e.g. by including only unique elements.
+   */
+  sampleWithoutReplacement(sampleSize = 10, cornerCases: FluentPick<A>[] = []): FluentPick<A>[] {
```

Comment:
nit: `sampleUnique` is more concise and more user friendly for those without statistical background. I'm ok either way though.

Reply:
@hugoferreira Should we use _unique_ or _withoutReplacement_?

Reply:
I'm ok with `sample` and `sampleUnique` 👍